stan/math/prim/mat.hpp

```diff
@@ -229,6 +229,8 @@
 #include <stan/math/prim/mat/prob/wishart_log.hpp>
 #include <stan/math/prim/mat/prob/wishart_rng.hpp>
 
+#include <stan/math/prim/mat/vectorize/apply_scalar_unary.hpp>
```

Comment:
(fix later) We don't need blank lines between includes.  And where possible, I'd prefer to just alphabetize (using lexicographic ordering on the path) everything.